acasclient/acasclient.py

```diff
@@ -1761,4 +1761,24 @@ def setup_items(self, item_type, items):
             raise ValueError("item_type must be one of {}".format(allowed_types))
         resp = self.session.post("{}/api/setup/{}".format(self.url, item_type), json=items)
         resp.raise_for_status()
+        return resp.json()
+
+    def get_lot_dependencies(self, lot_corp_name):
+        """Get lot dependencies for a lot by corp name
```

Comment:
It's not clear what data structure this returns. Can you expand this docstring and add some info about the data structure that's expected back?

Reply:
Added a returns data structure example: https://google.github.io/styleguide/pyguide.html#doc-function-raises